pkg/monitor/cluster/machineconfigpool.go

```diff
@@ -41,7 +41,8 @@ func (mon *Monitor) getNodeCounts(ctx context.Context) (int64, error) {
 	return int64(len(ns.Items)), nil
 }
 
-/***
+/*
+**
```

Comment:
nit: just remove `**`?

Reply:
Or switch to line comments since this is the only place that uses block comments (besides a few `doc.go` files).